hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx

```diff
@@ -117,7 +123,8 @@ cdef class AvellanedaMarketMakingStrategy(StrategyBase):
         self.c_add_markets([market_info.market])
         self._ticks_to_be_ready = max(volatility_buffer_size, trading_intensity_buffer_size)
         self._avg_vol = InstantVolatilityIndicator(sampling_length=volatility_buffer_size)
-        self._trading_intensity = TradingIntensityIndicator(trading_intensity_buffer_size)
+        self._trading_intensity_buffer_size = trading_intensity_buffer_size
+        self._trading_intensity = None      # Wait for network to be initialized
```

Comment:
Because we need to pass it an order book.

Without waiting:

![image](https://user-images.githubusercontent.com/7762229/165624850-243430de-0c4e-4af7-bcd1-6609cd5d5ad3.png)

2022-04-27 22:23:51,930 - 77972 - hummingbot.client.hummingbot_application - ERROR - Unknown error during initialization.
Traceback (most recent call last):
  File "/Users/matejhorvath/github/hummingbot/hummingbot/hummingbot/strategy/avellaneda_market_making/start.py", line 83, in start
    self.strategy.init_params(
  File "hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx", line 129, in hummingbot.strategy.avellaneda_market_making.avellaneda_market_making.AvellanedaMarketMakingStrategy.init_params
    order_book=self.market_info.order_book,
  File "/Users/matejhorvath/github/hummingbot/hummingbot/hummingbot/strategy/market_trading_pair_tuple.py", line 23, in order_book
    return self.market.get_order_book(self.trading_pair)
  File "hummingbot/connector/exchange/paper_trade/paper_trade_exchange.pyx", line 1111, in hummingbot.connector.exchange.paper_trade.paper_trade_exchange.PaperTradeExchange.get_order_book
    return self.c_get_order_book(trading_pair)
  File "hummingbot/connector/exchange/paper_trade/paper_trade_exchange.pyx", line 1021, in hummingbot.connector.exchange.paper_trade.paper_trade_exchange.PaperTradeExchange.c_get_order_book
    raise ValueError(f"No order book exists for '{trading_pair}'.")
ValueError: No order book exists for 'ETH-USDT'.
2022-04-27 22:23:51,938 - 77972 - hummingbot.client.hummingbot_application - INFO - start command initiated.
2022-04-27 22:23:52,019 - 77972 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: 'NoneType' object has no attribute 'closing_time'
Traceback (most recent call last):
  File "/Users/matejhorvath/github/hummingbot/hummingbot/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/Users/matejhorvath/github/hummingbot/hummingbot/hummingbot/client/command/start_command.py", line 45, in _run_clock
    await clock.run()
  File "hummingbot/core/clock.pyx", line 86, in run
    await self.run_til(float("nan"))
  File "hummingbot/core/clock.pyx", line 101, in run_til
    child_iterator.c_start(self, self._current_tick)
  File "hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx", line 568, in hummingbot.strategy.avellaneda_market_making.avellaneda_market_making.AvellanedaMarketMakingStrategy.c_start
    self._execution_state.time_left = self._execution_state.closing_time
AttributeError: 'NoneType' object has no attribute 'closing_time'
2022-04-27 22:23:52,543 - 77972 - hummingbot.core.rate_oracle.rate_oracle - INFO - Network status has changed to NetworkStatus.CONNECTED. Starting networking...
2022-04-27 22:24:01,120 - 77972 - hummingbot.client.hummingbot_application - INFO - stop command initiated.